samples/client/others/typescript/builds/null-types-simple/types/ObservableAPI.ts

```diff
@@ -1,5 +1,5 @@
 import { ResponseContext, RequestContext, HttpFile, HttpInfo } from '../http/http';
-import { Configuration, ConfigurationOptions } from '../configuration'
+import { Configuration, ConfigurationOptions, mergeConfiguration } from '../configuration'
```

Comment:
is there a simple way for us to detect if it is used or not and not produce this diff if it is not used?

Reply:
This function is used always (if there is at least a single operation),  so I think it makes sense to always have the import.